pydicom/data/base.py

```diff
@@ -0,0 +1,51 @@
+"""pydicom data manager"""
+#
+# Copyright (c) 2008-2012 Darcy Mason
+# Copyright (c) 2017 pydicom AUTHORS
+# This file is part of pydicom, released under a modified MIT license.
+#    See the file LICENSE included with this distribution, also
+#    available at https://github.com/pydicom/pydicom
+#
+
+import fnmatch
+import os
+
+from .utils import get_files
+
+
+def get_datadir():
+    return os.path.abspath(os.path.dirname(__file__))
+
+
+def get_testdata_base():
+    return os.path.join(get_datadir(), 'test_files')
+
+
+def get_charset_base():
+    return os.path.join(get_datadir(), 'charset_files')
+
+
+'''
+
+Data get functions.
+
+For each of the below, a complete list of files is returned.
+Note the distinction between "get" (returning a list) and
+what might be interpreted as reading in the files (load).
+These functions serve to only provide the paths.
+
+Optionally, the user can specify a pattern to filter by.
+
+'''
+
+
+def get_charset_files(pattern=None):
```

Comment:
I think its more readable like this (and we avoid the creation of functions):

```py
from pydicom.data import get_charset_base
from pydicom.utils import get_files

bla bla

get_files(get_charset_base())
```
I find it more readable.
Pushing it a bit further, do we need to write our own get_files? or is it enough to use glob? 
I mean, If I had written a `get_files` it would have been in order to encapsulate the filtering patterns. So that they are not accessible to the user. But if I end up exposing patterns then I would try to get it done with std library.